From bc3734051d762442bb977f675ff0cfaae927839a Mon Sep 17 00:00:00 2001
From: Vladimir Krivokapic <krivokapicvlada10@gmail.com>
Date: Sun, 5 Oct 2025 23:54:53 +0200
Subject: [PATCH] update gitignore

---
 .gitignore         |  4 ++++
 cbookKR/pointers.c | 11 +++++++++--
 2 files changed, 13 insertions(+), 2 deletions(-)

diff --git a/.gitignore b/.gitignore
index efae593..d7f524f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -13,6 +13,10 @@ dist/
 build/
 *.egg-info/
 bin/
+node_modules/
+tags
+ctags
+*.json
 
 .php-version
 ###> symfony/framework-bundle ###
diff --git a/cbookKR/pointers.c b/cbookKR/pointers.c
index d8c23ec..1b3e8a6 100644
--- a/cbookKR/pointers.c
+++ b/cbookKR/pointers.c
@@ -40,13 +40,20 @@ static int *int_copy(int *dest, int *src, int n)
     return dest;
 }
 
+static int sum(int a, int b){
+    return a + b;
+}
+
 int main(void)
 {
 
+    char buff[90];
+    printf("buff address: %p\nbuff[0] = %d\n", &buff, buff[0]);
     mystrcpy2(strB, strA);
     puts(strB);
-    mystrcpy(strC, strB);
-    puts(strC);
+    // puts(strB);
+    // mystrcpy(strC, strB);
+    // puts(strC);
 
     //int_copy(ys, xs, 4);
     //size_t sizey = sizeof(ys) / sizeof(ys[0]);
-- 
2.47.0

